docpages/make_a_bot/cmake.md

```diff
@@ -1,74 +1,87 @@
-\page buildcmake Building a Discord Bot using CMake/UNIX
+\page buildcmake Building a Discord Bot Using CMake (UNIX)
 
 ## 1. Toolchain
-Before compiling, you will need to install `cmake` on your system.
-To be sure that `cmake` is installed, you can type the following command:
-
-    $ cmake --version
-    cmake version 3.20.4
+Before compiling, you will need to install `cmake` on your system. To be sure that `cmake` is installed, you can type the following command:
 
+```bash
+$ cmake --version
+cmake version 3.22.1
+```
 
 ## 2. Create a CMake project
 
 In an empty directory, create the following files and directories:
 
-    - your_project/
-        |-- libs/
-        |-- src/
-            |-- main.cpp
-        |-- CMakeLists.txt
-
-
-In the `libs/` directory, clone D++ with: `git clone https://github.com/brainboxdotcc/DPP.git`
+```puml
```

Comment:
why not make these embedded graphviz?  like on threadmodel page? It will look so much nicer.

Reply:
Hm yeah good idea

Reply:
remember you can draw these in dia, then paste the text of the saved .dot file in